package.json

```diff
@@ -33,34 +33,34 @@
     "@types/fs-extra": "5.0.1",
     "@types/handlebars": "4.0.36",
     "@types/highlight.js": "9.12.2",
-    "@types/lodash": "4.14.104",
+    "@types/lodash": "4.14.106",
```

Comment:
Next time, avoid updating dependencies unnecessarially. It creates merge conflicts with other contributions.

Reply:
ok got it... I can revert it